### DISCUSSION TITLE
Time spent training the hific model

### DISCUSSION THREAD

**Discussion by zengming16** [tensorflow/compression#82] [Q&A]
Hello,
I'm a postgraduate from the Institute of Automation，Chinese Academy of Sciences.
I'm trying to using coco2014 from tfds to train the hific model as the README.md said. However, the first command 'python -m hific.train --config mselpips --ckpt_dir ckpts/mse_lpips --num_steps 1M --tfds_dataset_name coco2014' has taken me nearly 10 days to finish its work. I'm not sure whether it is normal. Please, can you tell me the time  you spent training your model?

Thank you in advance for your help.

**Comment by fab-jul** [comment]
Hi,
Hm, this seems somewhat excessive. What hardware did you use?
Fabian

**Reply by zengming16** [reply]
Hello，
Thanks for your attention. 
We used _56*Intel(R) Xeon(R) CPU E5-2683 v3 @ 2.00GHz_ and _MemTotal: 256GB_. 
Please,can you tell me the time you spent  training the model each command and what hardware you used as a reference?

Thank you in advance for your help.
